tasks.yaml

```diff
@@ -10,6 +10,7 @@ includes:
   - deploy: ./tasks/deploy.yaml
   - test: ./tasks/test.yaml
   - lint: ./tasks/lint.yaml
+  - common-setup: https://raw.githubusercontent.com/defenseunicorns/uds-common-tasks/v0.2.0/tasks/setup.yaml
```

Comment:
Sorry for another comment - might be good to update to latest tag and repo name here just so we don't forget down the line. Also tracking this with renovate?

Reply:
@mjnagel good call - this has been updated along with the renovate config